src/sentry/services/hybrid_cloud/user/serial.py

```diff
@@ -79,12 +79,7 @@ def serialize_rpc_user(user: User) -> RpcUser:
     else:
         orm_avatar = user.avatar.first()
         if orm_avatar is not None:
-            avatar = RpcAvatar(
-                id=orm_avatar.id,
-                file_id=orm_avatar.file_id,
-                ident=orm_avatar.ident,
-                avatar_type=orm_avatar.get_avatar_type_display(),
-            )
+            avatar = serialize_user_avatar(avatar=orm_avatar)
```

Comment:
If the avatar is serialized onto users, do we need a different RPC method? We could fetch the user with the avatar. :shrug:

Reply:
That's true and I did think about that, but figured it was worth cutting out the extra DB queries for the user and related entities. If we're not worried about the performance implication of this, I'm fine just using the get user RPC instead.

Reply:
I think having a smaller dedicated RPC makes sense. User includes a bunch of data we're not going to user here.